docs/source/gbd2017_models/risk_attributable_causes/vitamin_a_deficiency/index.rst

```diff
@@ -141,44 +149,148 @@ Restrictions
 Relative Risks
 ^^^^^^^^^^^^^^
 
-The affected causes with the Vitamin A deficiency cause in GBD 2017 include
+The causes affected by the Vitamin A Deficiency risk in GBD 2017 include
 :ref:`lower respiratory infections <2017_cause_lower_respiratory_infections>`,
-:ref:`diarrhoeal diseases <2017_cause_diarrhea>`, :ref:`measles
+:ref:`diarrhoeal diseases <2017_cause_diarrhea>`, and :ref:`measles
 <2017_cause_measles>`. The relative risks for these causes appear in Table 4 on
-p. 112 of [GBD-2017-Risk-Appendix-VAD]_.
+p. 112 of [GBD-2017-Risk-Appendix-VAD]_, copied here for reference:
+
+.. _gbd_2017_vad_relative_risk_table:
 
 .. list-table:: Pooled relative risks for risk-outcome pairs included in GBD 2017
-	:widths: 15 13 15 15
-	:header-rows: 1
-
-	* - Cause
-	  - GBD 2016 RR
-	  - GBD 2017 RR
-	  - Include in GBD 2017
-	* - Diarrhea
-	  - 1.6 (1.21 - 2.02)
-	  - 2.35 (2.17 - 2.54)
-	  - Yes
-	* - Measles
-	  - 2.4 (1.61 - 3.48)
-	  - 2.76 (2.01 - 3.78)
-	  - Yes
-	* - Lower Respiratory Infections (LRI)
-	  - 
-	  - 1.23 (1.03 - 1.48)
-	  - Yes
-	* - Meningitis
-	  - 
-	  - 3.2 (0.69 - 14.75)
-	  - No (not significant)
-	* - Malaria
-	  - 
-	  - 3.65 (2.23 - 5.97)
-	  - No (only one study)
-	  
+  :widths: 15 13 15 15
+  :header-rows: 1
+
+  * - Cause
+    - GBD 2016 RR
+    - GBD 2017 RR
+    - Include in GBD 2017
+  * - Diarrhea
+    - 1.6 (1.21 - 2.02)
+    - **2.35 (2.17 - 2.54)**
+    - Yes
+  * - Measles
+    - 2.4 (1.61 - 3.48)
+    - **2.76 (2.01 - 3.78)**
+    - Yes
+  * - Lower Respiratory Infections (LRI)
+    -
+    - **1.23 (1.03 - 1.48)**
+    - Yes
+  * - Meningitis
+    -
+    - 3.2 (0.69 - 14.75)
+    - No (not significant)
+  * - Malaria
+    -
+    - 3.65 (2.23 - 5.97)
+    - No (only one study)
+
+The above relative risks for GBD 2017 should be interpreted as rate ratios for
+the incidence rates of diarrhea, measles, and LRI.
+
 Vivarium Modeling Strategy
 --------------------------
 
+We will use an **exposure model** (or **prevalence-only model** or **propensity
+model**) for a vitamin A deficiency, in which each simulant is initialized with a "propensity" for vitamin A deficiency, and the simulant's vitamin A status is determined by comparing this
+propensity to the overall VAD exposure/prevalence in the population.
+Such
+propensity/exposure models have been used in Vivarium for other risk factors and
+risk-attributable causes, such as child stunting, :ref:`child wasting/PEM
+<2017_cause_pem>`, and :ref:`iron deficiency anemia
+<2017_cause_iron_deficiency>`.
+
+In more detail, the basic strategy is to initialize each simulant with a
+propensity score distributed uniformly in [0,1], then compare this propensity
+score with the (location/age/sex/year/intervention-status)-dependent prevalence
```

Comment:
Use exposure throughout.

Reply:
What do you mean?  Do you mean I should call this the "(location/age/sex/year/intervention-status)-dependent exposure distribution," which in this case happens to represent prevalence of VAD?

Reply:
Yeah, I think in a risk-based model, we should explicitly use exposure since the disease prevalence is derived from it.  As opposed to a PAF of 1 model of VitA in which the disease model was the source of truth and the risk exposure was derived from prevalence.

Reply:
Added some wording and a todo to reword everything to ensure consistent terminology.

I guess I was thinking about this from the GBD perspective - my impression is that they ran a DisMod model to get the cause data, and just copied the prevalence data over to the risk factor to get exposure.

Reply:
Makes sense.  As long as your consistent, it seems fine either way.